client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/AddActivity/LinkedTo/LinkedTo.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import LeadAutocompleteWrapper from './LeadAutocompleteWrapper/LeadAutocompleteWrapper';
+import * as styles from './LinkedTo.css';
+
+interface Props {
+
```

Comment:
redundant line

Reply:
we need new line before function in props